### PR TITLE
Add QSL Sizes to the Table and Rules

### PR DESCRIPTION
**Pull request by tjablin**

### CODE REVIEW

**Comment by christ1ne** [comment]
also updated the correct multi-stream latency targets.

**Comment by christ1ne** [comment]
the content is approved by the WG.

File: inference_rules.adoc [hunk 1]
```diff
@@ -194,17 +194,21 @@ The benchmark suite consists of the benchmarks shown in the following
...
 
 |===
-|Area |Task |Model |Dataset |Quality |Server latency constraint| Multi-Stream latency constraint
-|Vision |Image classification |Resnet50-v1.5 |ImageNet (224x224) | 99% of FP32 (76.46%) | 15 ms | 50 ms
-|Vision |Image classification |MobileNets-v1 224 |ImageNet  (224x224) | 99% of FP32 (71.68%) | 10 ms | 66 ms
-|Vision |Object detection |SSD-ResNet34 |COCO (1200x1200) | 99% of FP32 (0.20 mAP) | 100 ms | 50 ms
-|Vision |Object detection |SSD-MobileNets-v1 |COCO (300x300) | 99% of FP32 (0.23 mAP) | 10 ms | 50 ms
-|Language |Machine translation |GMNT |WMT16 | 99% of FP32 (23.9 BLEU) | 250 ms | 100 ms
+|Area |Task |Model |Dataset |QSL Size |Quality |Server latency constraint| Multi-Stream latency constraint
+|Vision |Image classification |Resnet50-v1.5 |ImageNet (224x224) | 3072 | 99% of FP32 (76.46%) | 15 ms | 50 ms
```
Comment (psyhtest) [inline inference_rules.adoc]: 500?

**Comment by christ1ne** [comment]
still need to decide 200 vs 500.

**Comment by tjablin** [comment]
You mean 500 vs 3072?

**Comment by christ1ne** [comment]
per WG agreement, Christine will merge.

**Pull request merged**